uaclient/config.py

```diff
@@ -1024,11 +1076,33 @@ def process_config(self):
                 ).format(prop)
                 raise exceptions.UserFacingError(error_msg)
 
+        if (self.global_apt_http_proxy or self.global_apt_https_proxy) and (
+            self.ua_apt_http_proxy or self.ua_apt_https_proxy
+        ):
+            # Should we unset the config values?
+            raise exceptions.UserFacingError(
+                "Error: Setting global apt proxy and ua scoped apt proxy at"
+                " the same time is unsupported."
+                " Cancelling config process operation."
```

Comment:
Let's either create a special Exception here or move the message to the `messages` module